lib/jira/commands/assign.rb

```diff
@@ -2,8 +2,9 @@ module Jira
   class CLI < Thor
 
     desc "assign", "Assign a ticket to a user"
+    method_option :assignee, aliases: "-a", type: :string, default: nil, lazy_default: "auto", banner: "ASSIGNEE"
```

Comment:
as i mentioned in your other pull request, id prefer if the aliases were more explicit `--assignee` until we have a common set of options we can default single-letter options to
not sure what your use case is, but non-interactive mode is usually used for scripts. would having it be more explicit work for your use case?